puppet/modules/quickstack/manifests/pacemaker/nosql.pp

```diff
@@ -92,7 +92,7 @@
     Exec['all-nosql-nodes-are-up'] ->
 
     quickstack::pacemaker::resource::service {'mongod':
-      options        => 'start timeout=10s',
+      options        => 'op start timeout=10s',
```

Comment:
Looking at the code that would build this in puppet-pacemaker, it appears this would net a pcs create command that contains 'op' twice, which I think at best is not ideal, and at worst, will result in more incorrect behavior.  The command it builds is:
```
cmd = 'resource create ' + @resource[:name] + ' ' + @resource[:resource_type] + ' ' + @resource[:resource_params] + ' op monitor interval=' + @resource[:interval]
```
options gets passed in as @resource[:resource_params] above.  Perhaps instead we should move nosql to use quickstack::pacemaker::resource::generic instead, and pass in 
```
operation_opts   => "start timeout=120s"
```
as we do for neutron-server?  This looks more like lines 48 & 50 from the scenario:
https://github.com/beekhof/osp-ha-deploy/blob/master/pcmk/mongodb.scenario#L48